libpromises/cf3.defs.h

```diff
@@ -67,6 +67,7 @@
 #define CF_BLOWFISHSIZE 16
 #define CF_MAXVARSIZE 1024
 #define CF_MAXSIDSIZE 2048      /* Windows only: Max size (bytes) of security identifiers */
+#define CF_MAXFRAGMENT 19       /* abbreviate long promise names to 2*MAXFRAGMENT+3 */
```

Comment:
because it needs to take MAX_FRAGMENT chars from the beginning of the promise string, and MAX_FRAGMENT from the end, and insert `...` in the middle.

I think it's better to define MAX_FRAGMENT this way than `assert((MAX_FRAGMENT - 3) / 2 > 0)` later on.